.eslintrc.cjs

```diff
@@ -117,7 +117,5 @@ module.exports = {
     },
   },
   parser: "@typescript-eslint/parser",
-  ignorePatterns: [
-
-  ],
+  ignorePatterns: ["starters/**"],
```

Comment:
The rules of the parent `eslintrc` were being applied to the starters templates but the templates already have their own `eslintrc.json` file.